packages/jumpstarter-driver-flashers/oci_bundles/build_bundle.sh

```
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# oras login quay.io -u mangelajo
+
+FLASHER_OCI_CONTAINER="${1:-quay.io/jumpstarter-dev/jumpstarter-flasher-test:latest}"
+BUNDLE_FILES=${2:-"./test/"}
+
+echo "Building and pushing ${FLASHER_OCI_CONTAINER}"
+
+set -x
+
+cd "${BUNDLE_FILES}"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)